fornax.py

```diff
@@ -245,18 +245,19 @@ def process_data_info(self):
                     # TO ACCESS REGION-RESTRICTED DATA ANONYMOUSLY.
                     # -----------------------
                     # Attempting anonymous access:
-                    msg = 'Accessing region data anonymously ...'
-                    s3_config = botocore.client.Config(signature_version=botocore.UNSIGNED)
-                    s3_resource = boto3.resource(service_name='s3', config=s3_config)
-                    accessible, message = self.is_accessible(s3_resource, aws_info['bucket'], aws_info['path'])
-                    if accessible:
-                        break
-                    message = f'  - {msg} {message}.'
-                    messages.append(message)
+                    if data_access == 'region': 
+                        msg = f'Accessing {data_access} data anonymously ...'
```

Comment:
why does `msg` need to be defined here, it's always the same, so could just go into L255 directly.

Reply:
It is defined here because if `accessible` is `True`, the while loop exits at L253 and L255 won't be reached. Maybe there are better ways of doing this.